README.md

```diff
@@ -164,6 +164,50 @@ In order to use "Unbreakable Selenium", you just need to wrap your usual driver
  re = new RecheckWebImpl();
 ```
 
+## Usage of rehub
+
+***rehub*** is a browser-based business intelligence platform that includes a repository service for simplified test automation.
+
+To upload your test reports to [***rehub***](https://retest.de/rehub/), you need to take the following steps:
+
+1. Setup your ***retest*** [account](https://sso.prod.cloud.retest.org/auth/realms/customer/protocol/openid-connect/auth?response_type=code&client_id=garkbit&redirect_uri=http%3A%2F%2Fgarkbit.prod.cloud.retest.org%2Fsso%2Flogin&state=512ba44f-b51e-460b-80af-fc0964f1909e&login=true&scope=openid). You will receive a 14-day trial.
+
+2. To enable the upload you have to modify your `setUp()` method in your tests. There are two possibilities:
+
+ - by setting up the `REHUB_REPORT_UPLOAD_ENABLED` system property
+    
+```java
+driver = new ChromeDriver();
+re = new RecheckImpl();
+System.setProperty( Properties.REHUB_REPORT_UPLOAD_ENABLED, "true" );
+```
+
+ - or by modify the `RecheckImpl` constructor
+
+```java
+driver = new ChromeDriver();
+re = new RecheckImpl( RecheckOptions.builder().reportUploadEnabled( true ).build() );
+```
+
+If you execute the test locally and the configuration was successful, your browser will pop up and you will be prompted to login, then you can find your test reports on the ***rehub garkbit***.
```

Comment:
We need to pay attention to what marketing names are present vs what names we use internally. Do we really want to market the name `garkbit`? @roesslerj